src/core/src/package_managers/PackageManager.py

```diff
@@ -61,6 +61,13 @@ def refresh_repo(self):
         """Resynchronize the package index files from their sources."""
         pass
 
+    def refresh_repo_safely(self):
+        """Resynchronize the package index files from machine sources."""
+        try:
+            self.refresh_repo()
+        except Exception as error:
+            self.composite_logger.log_error("Error in refreshing cache from machine sources. [Error={0}]".format(repr(error)))
```

Comment:
log_debug is fine - this error is not customer facing and doesn't need to be reported to them. What we are doing here is best effort.

Reply:
Agree with the log type. What is your view on reporting this error in status blob? I'm still debating over. It gets logged within ZypperPkgMgr currently. Not the other PkgMgrs

Reply:
That's ok, this is just a quiet in & out operation.. 

The zypper flow is extremely unlikely to ever trigger something new here that hasn't been triggered before and been acted on. 

But - unrelated to this PR, 'Setting force_reboot flag to True' in Zypper needs to be a log_debug too. It's not a customer-facing message. Please make the change here too.

Reply:
Addressed